phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -420,8 +421,11 @@ public ConnectionQueryServicesImpl(QueryServices services, ConnectionInfo connec
             list.add(queue);
         }
         connectionQueues = ImmutableList.copyOf(list);
+
         // A little bit of a smell to leak `this` here, but should not be a problem
```

Comment:
we can remove this comment now/

Reply:
Leaving until https://issues.apache.org/jira/browse/PHOENIX-5244 is resolved.